canonical-web-and-design/ubuntu.com
# Remove public sector event strip

## Done

- Removed the  "Meet our Public Sector team at AWS Public Sector Summit" from /gov, /aws and /aws/pro

## QA

- View the site in your web browser at: https://ubuntu-com-11686.demos.haus/gov
- See that the "Meet our Public Sector team at AWS Public Sector Summit" has gone (compare it to live)
- Visit https://ubuntu-com-11686.demos.haus/aws and https://ubuntu-com-11686.demos.haus/aws/pro and verify they also don't show the event strip

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/5473, https://github.com/canonical-web-and-design/web-squad/issues/5474, https://github.com/canonical-web-and-design/web-squad/issues/5475
